fast_neural_style/neural_style/neural_style.py

```diff
@@ -229,28 +224,42 @@ def main():
                                  help="path for saving the output image")
     eval_arg_parser.add_argument("--model", type=str, required=True,
                                  help="saved model to be used for stylizing the image. If file ends in .pth - PyTorch path is used, if in .onnx - Caffe2 path")
-    eval_arg_parser.add_argument("--cuda", type=int, default=False,
-                                 help="set it to 1 for running on cuda, 0 for CPU")
+    eval_arg_parser.add_argument('--no-cuda', action='store_true', default=False,
```

Comment:
I think you will likely cause CI break on pytorch side with this change. Better to revert it.